packages/code-studio/src/main/AppInit.tsx

```diff
@@ -167,7 +167,7 @@ const AppInit = (props: AppInitProps) => {
         dh.IdeConnection.HACK_CONNECTION_FAILURE,
         event => {
           const { detail } = event;
-          log.error('Connection failure', detail);
+          log.error('Connection failure', `${detail}`);
```

Comment:
Should probably do `JSON.stringify` if `detail` is an object, otherwise we'll just get `'[object Object]'`
```suggestion
          log.error('Connection failure', `${JSON.stringify(detail)}`);
```